File: site-estatico/models/usuario.js
"use strict";

module.exports = (sequelize, DataTypes) => {
  let usuario = sequelize.define(
    "usuario",
    {
      id_usuario: {
        field: "id_usuario",
        type: DataTypes.INTEGER,
        primaryKey: true,
        autoIncrement: true,
      },
      nome: {
        field: "nome",
        type: DataTypes.STRING,
        allowNull: false,
      },
      email: {
        field: "email",
        type: DataTypes.STRING,
        allowNull: false,
      },
      senha: {
        field: "senha",
        type: DataTypes.STRING,
        allowNull: false,
      },
      cargo: {
        field: "cargo",
        type: DataTypes.ENUM,
        allowNull: false,
      },
      fk_empresa: {
        field: "fk_empresa",
        type: DataTypes.STRING,
        allowNull: false,
      },
      fk_supervisor: {
        field: "fk_empresa",
        type: DataTypes.STRING,
        allowNull: true,
      },
    },
    {
      tableName: "usuario",
      freezeTableName: true,
      underscored: true,
      timestamps: false,
    }
  );

  return usuario;
};

File: Apps/Java/clientJava/src/main/java/com/mycompany/projeto/artefato/sprint/entidades/Usuario.java
package com.mycompany.projeto.artefato.sprint.entidades;

public class Usuario {
    private Integer id_usuario;
    private String nome;
    private String email;
    private String senha;
    private String cargo;
    private String token;
    private String fk_empresa;
    private Integer fk_supervisor;

    public Usuario(Integer id_usuario, String nome, String email, String senha,
            String cargo, String token, String fk_empresa, Integer fk_supervisor) {
        this.id_usuario = id_usuario;
        this.nome = nome;
        this.email = email;
        this.senha = senha;
        this.cargo = cargo;
        this.token = token;
        this.fk_empresa = fk_empresa;
        this.fk_supervisor = fk_supervisor;
    }
}
